### PR TITLE
Boilerplate, set up celery and minimal task

### PR DESCRIPTION
**Pull request by amercader**
@roll I've booted up the package with the basic stuff from [oky-py](https://github.com/okfn/oki-py). The main difference is having the requirements separate on a `requirements.txt`, which I think it's more convenient for a standalone app. Actually I'm pinning the actual requirements in `requirement.in` and using [pip-tools](https://github.com/nvie/pip-tools) to generate `requirements.txt` which has given me great results in the past.

Apart from that I created a minimal task and helper that should make easier to tests things.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import os
```
Comment (roll) [inline setup.py]: Why do we need all this setup.py, setup.cfg. tox etc stuff? I suppose it's only for libs.

Comment (amercader) [inline setup.py]: You need a `setup.py` for development and installation no? And tox is for running the tests, regardless if it's an app or a lib

File: LICENSE.md [hunk 2]
```diff
@@ -1,35 +1,37 @@
-                    GNU AFFERO GENERAL PUBLIC LICENSE
-                       Version 3, 19 November 2007
+GNU Affero General Public License
```
Comment (pwalsh) [inline LICENSE.md]: yes, following on from historical decisions, MIT for libs, and AGPL for apps. I'm considering changing everything to MIT, but that is a different discussion for another time.

**Comment by roll** [comment]
`requirements.in` is a cool thing)

**Comment by amercader** [comment]
@roll can you approve if happy with the rest? we can change the license later

**Pull request merged**